.github/workflows/unit_test_ci.yaml

```diff
@@ -0,0 +1,53 @@
+name: unit test ci
+
+on:
+  push:
+    branches:
+      - '*'
+
```

Comment:
The workflow is not picked up by Github actions because there is no trigger for a pull request.

Reply:
alright will add it now thanks